setup.py

```diff
@@ -19,19 +19,19 @@
 
 
 setup(
-    name="ga4gh_dos_schemas",
+    name="ga4gh_drs_schemas",
```

Comment:
I think this will break the PyPI release. We need to configure a new package on PyPI before merging this, especially since a version bump would result in a deploy

Reply:
Yep this will.  @natanlao and I discussed setting up a GA4GH Cloud Work Stream PyPI account and will transition to this during the hackathon next week.  We will hold off on any release/push to master until it's fixed .